1-WebApp-OIDC/1-1-MyOrg/Startup.cs

```diff
@@ -32,16 +34,19 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             // Sign-in users with the Microsoft identity platform
-            services.AddMicrosoftIdentityPlatformAuthentication(Configuration);
+            //services.AddSignIn(Configuration);
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                .AddSignIn("AzureAD", Configuration, options => Configuration.Bind("AzureAD", options));
```

Comment:
`AddSignIn` expected an `Action<OpenIdConnectOptions>` so it is easier to extend the configurations done by the extension library.